src/pages/ExpandedPost.tsx

```diff
@@ -2,22 +2,30 @@ import { Box, Divider } from "@mui/material";
 import { Stack } from "@mui/system";
 import ExpandedPostItem from "../components/Posts/ExpandedPostItem";
 import ComposeReply from "../components/Posts/ComposeReply";
-import ExpandedPostList from "../components/Posts/ExpandedPostReplies";
+import ExpandedPostReplies from "../components/Posts/ExpandedPostReplies";
+import { useAppSelector } from "../state/hooks";
 
 const styles = {
   root: {
     width: "100%",
   },
   postListContainer: { padding: 0, width: "50%" },
+  divider: { paddingBottom: 3 },
```

Comment:
Does this get used?

Reply:
It wasn't originally used, but I ended up using it just now to help fix some padding issues on `ComposeReply.tsx` component within the reply modal which consequentially affected `ExpandedPost.tsx`